cf-serverd/cf-serverd.c

```diff
@@ -61,7 +63,9 @@ int main(int argc, char *argv[])
         policy = LoadPolicy(ctx, config);
     }
 
+    GenericAgentPostLoadInit(ctx);
```

Comment:
Beware! There is a separate implementation of this in ``libcfexecd/win_service_exec.c``, specifically for the Windows service framework. You need to make the change there as well.